lang_qc/db/helper/wells.py

```diff
@@ -147,12 +147,23 @@ class PacBioPagedWellsFactory(PagedStatusResponse):
         description="A SQLAlchemy Session for the ml warehouse database",
     )
 
+    # For MySQL it's OK ot use case-sensitive comparison operators since
+    # its string comparisons are string-insensitive.
     FILTERS: ClassVar = {
         QcFlowStatusEnum.ON_HOLD.name: (QcStateDict.state == "On hold"),
         QcFlowStatusEnum.QC_COMPLETE.name: (QcState.is_preliminary == 0),
         QcFlowStatusEnum.IN_PROGRESS.name: and_(
             QcState.is_preliminary == 1, QcStateDict.state != "On hold"
         ),
+        QcFlowStatusEnum.ABORTED.name: or_(
+            PacBioRunWellMetrics.well_status.like("Abort%"),
```

Comment:
Does the warehouse data need us to be this flexible? It smells bad (and has performance impact).